catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducer.java

```diff
@@ -55,26 +51,9 @@
 
     private static final int DEFAULT_FILE_BACKED_OUTPUT_STREAM_THRESHOLD = 1 * MB;
 
-    private ContentEndpoint endpoint;
-
-    private static final String FILE_ABSOLUTE = "CamelFileAbsolute";
-
-    private static final String FILE_ABSOLUTE_PATH = "CamelFileAbsolutePath";
+    private static final String ATTRIBUTE_OVERRIDES_KEY = "AttributeOverrides";
```

Comment:
This should be accessible from CatalogFramework so we don't have to define this twice. probably makes sense for the `Constants` class in catalog-core-api